qiskit_ibm_runtime/estimator.py

```diff
@@ -19,21 +19,117 @@
 
 # TODO import BaseEstimator and EstimatorResult from terra once released
 from .qiskit.primitives import BaseEstimator, EstimatorResult
-from .ibm_runtime_service import IBMRuntimeService
+from .exceptions import IBMInputValueError
+from .ibm_backend import IBMBackend
+from .qiskit_runtime_service import QiskitRuntimeService
 from .runtime_session import RuntimeSession
 
 
 class Estimator(BaseEstimator):
-    """Estimator primitive."""
+    """Class for interacting with Qiskit Runtime Estimator primitive service.
+
+    Qiskit Runtime Estimator primitive service estimates expectation values of quantum circuits and
+    observables.
+
+    Estimator can be initialized with the following parameters.
+
+    * circuits: a (parameterized) :class:`~qiskit.circuit.QuantumCircuit` or
+        a list of (parameterized) :class:`~qiskit.circuit.QuantumCircuit`.
+
+    * observables: a list of :class:`~qiskit.quantum_info.SparsePauliOp`.
+
```

Comment:
The source of the changes in this file (and the relation to the scope of the PR) is not clear to me. Is that copied from [qiskit-terra](https://github.com/Qiskit/qiskit-terra/tree/main/qiskit/primitives)

Reply:
Estimator becomes the primary interface for users instead of IBMEstimator. Issue description has more details.

Reply:
This is copied from IBMEstimator class.